src/main/resources/com/hcl/appscan/jenkins/plugin/scanners/StaticAnalyzer/config.jelly

```diff
@@ -4,8 +4,8 @@
 © Copyright IBM Corporation 2016.
 LICENSE: Apache License, Version 2.0 https://www.apache.org/licenses/LICENSE-2.0
 -->
-    <f:optionalBlock title="${%label.rescan}" name="rescan" checked="${instance.isRescan()}" inline="true">
-        <f:entry title="${%label.scan.id}" field="scanId">
+    <f:optionalBlock title="${%label.rescan}" name="rescan" checked="${instance.isRescan()}" help="/plugin/appscan/help/rescan.html" inline="true">
```

Comment:
@vishalhcl-5960 Why can't we apply the approach using help-<property>.jelly? Wouldn't that work in this case?"

Reply:
Kripa, We can use that approach too.

Reply:
Kripa, I will be using the html-help files as the rescan function is common across all scan types so with the help-.jelly approach i have to add the jelly files & resource bundles for all 3 types.